mrp_production_category_notes/models/mrp_production.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Angel Rivas <angel.rivas@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    @api.model
+    def _get_category_notes(self, product):
+        category = product.categ_id
+        while category:
+            if category.mrp_notes:
+                return category.mrp_notes
+            category = category.parent_id
+        return None
+
+    @api.model
+    def create(self, vals_list):
+        records = super(MrpProduction, self).create(vals_list)
+
+        for rec in records:
+            if rec.product_id:
+                mrp_notes = self._get_category_notes(rec.product_id)
+
+                if mrp_notes:
+                    rec.notes = mrp_notes
```

Comment:
Instead of replacing the value of notes, the value in mrp_notes should be added to the notes field, so if some information has been already added when the manufacturing order is created, it will be kept.